### DISCUSSION TITLE
TOC File Limit

### DISCUSSION THREAD

**Discussion by tparency** [CMSgov/price-transparency-guide#492] [Q&A] [answered]
We're splitting files into smaller chunks for our initial MRF TOC publication. Does CMS have a limit on how many files can be listed in a TOC?

**Comment by shaselton-usds** [comment, marked as answer]
At this point there isn't a limit. 

Just to be sure, these smaller chunks are subsets of the total file and when a consumer combines all of them from the TOC, it will be the complete file, correct?

How many files are you looking at at breaking down and of what types?

Thanks!
